script.js

```diff
@@ -153,40 +172,22 @@ button.addEventListener('click', function(){
                     var newDateFormat = new Date(alltimeValue[i] * 1000);
```

Comment:
As GitHub does not allow to comment unchanged lines, this is about the line with `querySelectorAll` above: it would be nice to get the checked elements once and share them.

Reply:
It is in function that used as first part of a fetch. So it takes ones and shared to preparing the data for the plotting and inner into HTML